src/logging.rs

```diff
@@ -3,30 +3,30 @@ use std::sync::Once;
 
 use log::{info, LevelFilter};
 
-use crate::Configuration;
+use crate::settings::Settings;
 
 static INIT: Once = Once::new();
 
-pub fn setup_logging(cfg: &Configuration) {
-    let level = config_level_or_default(&cfg.log_level);
+pub fn setup_logging(settings: &Settings) {
+    let level = settings_level_or_default(&settings.log_level);
 
     if level == log::LevelFilter::Off {
         return;
     }
 
     INIT.call_once(|| {
-        stdout_config(level);
+        stdout_settings(level);
```

Comment:
hi @da2ce7, in this case, "config" is a verb. The function `stdout_config` sets up the logging. I was tempted to rename it to `redirect_logging_to_stdout` or something like that.